lib/app_funcs.sh

```diff
@@ -16,7 +16,7 @@ function copy_hex() {
   mkdir -p ${build_path}/.hex
 
   # hex file names after elixir-1.1 in the hex-<version>.ez form
-  full_hex_file_path=$(ls -t ${HOME}/.mix/archives/hex-*.ez | head -n 1)
```

Comment:
It might.  I'll take at look at some point, and, in the meanwhile, use my fork for Outpost.